### ISSUE TITLE
authentications service misses some dependencies

### ISSUE THREAD

**Issue by andreastedile** [carlocorradini/graphmarket#155]
When starting the "authentications" service with NODE_ENV=development, we have an error:

Bootstrap error: Postgres package has not been found installed. Try to install it: npm install pg --save

**Comment by carlocorradini** [comment]
Manca la dipendenza pg ( il driver)

**Comment by andreastedile** [comment]
Suggerisci
```
npx lerna add pg --scope @graphmarket/service-authentications
```
?

**Comment by carlocorradini** [comment]
Yep

**Comment by andreastedile** [comment]
Fatto. Prossimo errore:
```
authentications_1    | 2021-01-27 21:16:48:419 ERROR 	Bootstrap error: Cannot use cache because ioredis is not installed. Please run "npm i ioredis --save".
```
Procedo ad installarlo

**Comment by carlocorradini** [comment]
Si stessa cosa ma con ioredis

**Comment by andreastedile** [comment]
Fixed via commit c3eef635
